CHANGELOG.md

```diff
@@ -60,6 +60,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Added `nix::ptrace` on all Linux-kernel-based platforms
   [#624](https://github.com/nix-rust/nix/pull/624). Previously it was
   only available on x86, x86-64, and ARM, and also not on Android.
+- Fixed missing public fields on `ucred` and renamed it to `UserCredentials`
```

Comment:
This is incorrect. Better to say "Renamed `ucred` to `UserCredentials` and exposed fields through additional methods." Also, this should be moved to the `Changed` section. This is a feature addition & API change rather than a bugfix.